generate-release-notes.py

```diff
@@ -44,14 +71,15 @@ def extract_release_notes(issue):
     # Look for a release note section in the body.
     matches = None
     if issue.body:
-        matches = re.findall(r'```release-note(.*?)```', issue.body, re.DOTALL)
+        matches = re.findall(r"```release-note(.*?)```", str(issue.body), re.DOTALL)
```

Comment:
Is the `str()` call necessary? What type is `issue.body` normally?

Reply:
it is necessary